src/wx/wxvbam.cpp

```diff
@@ -185,6 +185,27 @@ wxString wxvbamApp::GetAbsolutePath(wxString path)
     return path;
 }
 
+#ifndef NO_ONLINEUPDATES
+#include "winsparkle.h"
+#include "version.h"
+
+static void init_check_for_updates()
+{
+#ifdef __WXMSW__
+    std::wstring version;
+#ifndef FINAL_BUILD
+    version = _(VERSION);
+#else
+    version = wxT("0");
+#endif // FINAL_BUILD
+    //win_sparkle_set_appcast_url("https://github.com/visualboyadvance-m/visualboyadvance-m/data/appcast.xml");
+    win_sparkle_set_appcast_url("https://github.com/visualboyadvance-m/visualboyadvance-m/tree/update-checker/data/appcast.xml");
```

Comment:
@ZachBacon This is for testing only, but where should we save this file for us?

Reply:
visualboyadvance-m/data/appcast.xml seems alright with me, but so does the current location, as long as the updater can find it really.